src/main/java/frc/robot/subsystems/Drivetrain.java

```diff
@@ -285,10 +285,10 @@ public void updatePoseEstimator() {
    * 
    * @param visionRobotPose Pose of robot as calculated by the vision system
    */
-  public void addVisionMeasurement(Pose2d visionRobotPose) {
+  public void addVisionMeasurement(Pose2d visionRobotPose, double timestampSeconds) {
     poseEstimator.addVisionMeasurement(
         visionRobotPose,
-        Timer.getFPGATimestamp());
+        timestampSeconds);
```

Comment:
assuming you're only calling this method with valid pose data, does this change only matter for accuracy beyond +/- 20ms (50hz)

Reply:
i agree. the theoretical worst delta is 20ms (if i understand this correctly) and that could cause noticeable error